JavaScriptSPA/apiConfig.js

```diff
@@ -0,0 +1,10 @@
+// The current application coordinates were pre-registered in a B2C tenant.
+const apiConfig = {
+    b2cScopes: ["https://fabrikamb2c.onmicrosoft.com/helloapi/demo.read"],
+    webApi: "https://fabrikamb2chello.azurewebsites.net/hello"
+  };
+
+// request to acquire a token for resource access
+const tokenRequest = {
```

Comment:
Why is request here if the file is named `apiConfig.js`?

Reply:
wouldn't it make sense to group the api configuration with the token scopes it accepts? would you prefer to move it into api.js instead?